pkg/core/resources/apis/mesh/validators.go

```diff
@@ -234,14 +239,14 @@ func ValidateHostname(path validators.PathBuilder, hostname string) validators.V
 	}
 
 	if strings.HasPrefix(hostname, "*.") {
-		if !DomainRegexp.MatchString(strings.TrimPrefix(hostname, "*.")) {
+		if !DNS1123SubdomainRegexp.MatchString(strings.TrimPrefix(hostname, "*.")) {
```

Comment:
Any reason not to use the regexp already in the k8s validaiton package?

Reply:
They are private in k8s